projects/saas-ui/src/app/main/commands/add-tenant.command.ts

```diff
@@ -1,7 +1,7 @@
 import {ApiService, IAdapter, PostAPICommand} from '@project-lib/core/api';
 
 import {IAnyObject} from '@project-lib/core/i-any-object';
-import {Tenant} from '../../../on-boarding/models';
+import {Tenant} from '../../on-boarding/models';
```

Comment:
why are we referring on-boarding module in main module 
move the common models and other code to shared

Reply:
Done